tscached.yaml

```diff
@@ -27,8 +27,9 @@ tscached:
         staleness_threshold: 10  # data up to this far in the past is "new"
 
     chunking:
-        chunk_length: 1800  # chunk on 30 minute intervals
+        chunk_length: 3600  # chunk on 1 hour intervals
         max_chunks: 6  # increase chunk size if more than this needed
+        thread_timeout: 30  # timeout on waiting for chunk threads to join
```

Comment:
I actually don't want to do this. If you use the raw KairosDB internally, it has a very high timeout: 1 minute if I recall correctly. We'd rather time out before that in the thread. 1 minute is necessary for long (over 3 hours) queries. It's configurable, so we can always make a different choice later :)